### ISSUE TITLE
REQUEST: Can you add a LICENSE and a README.md explaining basic usage?

### ISSUE THREAD

**Issue by Janik-Haag** [ianthehenry/nix-zshell#1]
It would be great if you could add a LICENSE and explain how to use this. I know you explained a bit of stuff in your [blogpost](https://ianthehenry.com/posts/how-to-learn-nix/nix-zshell/) but didn't really note down how someone else could use this. I assume to use it, one basically has to just build it and set the env var NIX_BUILD_SHELL to the script but still.

**Comment by ianthehenry** [comment]
Oh hi! I don't really think that you *should* use this repo -- I was pretty surprised by this issue that I even *had* a repo for this. I guess I made this to sync it to my VPS one day? I really can't remember creating this.

I don't plan on supporting or licensing or documenting this hack, and I can't guarantee that it even works anymore, but the thing you said is a correct description of how I used it two years ago.
